.circleci/config.yml

```diff
@@ -6,9 +6,11 @@ defaults: &defaults
       - run:
           name: Replace Auth0 test credentials
           command: |
-            sed -i 's/{CLIENT_ID}/'$AUTH0_TEST_CLIENT_ID'/g' $AUTH0_CFG
-            sed -i 's/{DOMAIN}/'$AUTH0_TEST_DOMAIN'/g' $AUTH0_CFG
-            sed -i 's/{CLIENT_SECRET}/'$AUTH0_TEST_CLIENT_SECRET'/g' $AUTH0_CFG
+            cd $AUTH0_CFG/src/main/resources
+            mv auth0.properties.example auth0.properties
```

Comment:
I'd try to maintain the old style as it's clearer and makes use of the variable defined below.

Wondering if this would work
```bash
mv "${AUTH0_CFG}.example" $AUTH0_CFG
```

Then you remove the `cd` line and can keep the remaining `sed` lines as they were, and you'd rollback the variable value change in line 59.

Reply:
👍 yeah I think that will work and makes it less confusing to have two env vars with the same value. Will update and try it, thanks!

Reply:
@lbalmaceda I pushed a new commit that does as you suggested.